.github/workflows/build-distribute.yml

```
@@ -0,0 +1,145 @@
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    environment: ${{ inputs.environment }}
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Cache NDK
+        uses: actions/cache@v3
```

actionlint (1.7.10)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)